addons/stock/migrations/10.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# © 2017 Trescloud <http://trescloud.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    
+    #field route_sequence
+    openupgrade.copy_columns(
+        env.cr, {
```

Comment:
Use variables to store the list of column renames, copies, and so on, the same as in https://github.com/OCA/OpenUpgrade/pull/839/files#diff-3cdc4348ce357d41a6908e42cadbca5aR8

Group also all the copies in one.